bin/travis

```diff
@@ -3,9 +3,9 @@
 set -e
 
 if [[ $TRAVIS_SCALA_VERSION == 2.11* ]] && [[ $SCALAZ_VERSION = 7.2* ]]; then
-  SBT_COMMAND=";coverage ;clean ;test ;makeSite ;coverageOff"
+  SBT_COMMAND="\"set scalazVersion in ThisBuild := \\\"7.2.1\\\"\" ;coverage ;clean ;test ;makeSite ;coverageOff"
```

Comment:
Could also s/7.2.1/$SCALAZ_VERSION/ so we don't have to maintain it on patch releases of scalaz.